contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/MarkdownEditor/ImagesMenu/ImagesMenu.vue

```diff
@@ -23,9 +23,9 @@
       <VCardText>
         <div class="body-1 mb-2 mx-2">
           <FileStatusText
-            v-if="uploadingChecksum"
+            v-if="uploadingId"
+            :id="uploadingId"
```

Comment:
Might be good to call this property `fileId` as id is a bit more of a vague term in this context

Reply:
Updated here and on the FileStatus component.